cxx/gendb.cc

```diff
@@ -30,6 +30,14 @@ GenDB::GenDB(std::mt19937* prng, const PCleanSchema& schema_,
   }
 }
 
+double GenDB::logp_score() const {
```

Comment:
Ha!  In my pull-request in progress, I have a dummy GenDB::log_score method with a TODO assigned to you to fill it out.

Reply:
:)